### DISCUSSION TITLE
Can we submit updates to the courseware?

### DISCUSSION THREAD

**Discussion by microsmith** [MicrosoftLearning/AZ400-DesigningandImplementingMicrosoftDevOpsSolutions#428] [General]
I.e. https://learn.microsoft.com/en-us/training/courses/az-400t00

How do we submit requested changes to the courseware?

**Comment by LuizMacedo** [comment]
Hi @microsmith! You can send directly to me luiz.macedo at microsoft.com or [from our learning pages ](https://learn.microsoft.com/en-us/training/support/troubleshooting#report-feedback-for-training). Thanks!
